### DISCUSSION TITLE
no burn

### DISCUSSION THREAD

**Discussion by smc4256** [arkypita/LaserGRBL#1302] [Q&A]
**Describe the bug** laser lights will not burn seems like not enough power plus everything is in reverse

**To Reproduce**

**Expected behavior**

**Screenshots**

**Hardware and software configuration (please complete the following information):**
 - Windows version
 - Engraver brand and model
 - Grbl version (i.e. 0.9j, 1.1f)
 - LaserGRBL version

**Additional context**

**Comment by gmmanonymus111** [comment]
It seems like a hardware issue to me rather than LaserGRBL issue.
By `everything is in reverse` you mean even laser power (like `S0` = 100%, `S255` = 0%), or only movement is reversed?
What type of PSU (power supply) do you use?
How old and what kind of laser do you have?
And next time, please fill the issue template, that can speed up the process of resolving any issues.
